src/views/NavBar.vue

```diff
@@ -0,0 +1,49 @@
+<template lang="pug">
+  .nav-bar
+    img.navbar-logo(src='../assets/wolox-logo.png')
+    button.logout-button(@click='logout')
+      | Logout
+
+</template>
+
+<script>
+import { deleteCurrentSession } from '@/services/auth'
+
+export default {
+  name: 'navBar',
```

Comment:
No need for the name in this component

Reply:
Why is it? :thinking:

Reply:
The only components in need of a name are the `views` since otherwise, you'll only see them as anonymous. Since you are importing the components with a name, you can omit this field in the components under the `components` folder

Reply:
Great!! Thank you very much for the explanation! :smiley::+1: